app/src/test/java/edu/uc/groupProject/topten/ListItemDataUnitTest.kt

```diff
@@ -93,4 +90,24 @@ class ListItemDataUnitTest {
         toStringTest = ListItem(0, "Avengers", "A movie about Batman", 100)
         assertTrue(toStringTest.toString() == "Avengers")
     }
+
+    @Test
+    fun ensureCorrectVoteCounts() {
```

Comment:
Added test for edge case of ListItems with bad vote counts

Reply:
Good.  Increase test coverage.